db/migrate/20250108170244_create_users.rb

```
@@ -0,0 +1,8 @@
+class CreateUsers < ActiveRecord::Migration[7.1]
+  def change
+    create_table :users do |t|
+
+      t.timestamps
```

rubocop (1.69.1)

[convention] Extra empty line detected at block body beginning.

(Layout/EmptyLinesAroundBlockBody)